src/modules/dashboard/components/Dialogs/ControlSafeDialog/TransactionTypesSection/TransferFundsSection.tsx

```diff
@@ -30,6 +39,19 @@ const MSG = defineMessages({
     id: `dashboard.ControlSafeDialog.ControlSafeForm.TransferFundsSection.userPickerPlaceholder`,
     defaultMessage: 'Select or paste a wallet address',
   },
+  balancesLoading: {
+    id: `dashboard.ControlSafeDialog.ControlSafeForm.TransferFundsSection.balancesLoading`,
```

Comment:
Can we update the ids here :)

Reply:
What part of the ids needs to be updated?

Reply:
`dashboard.ControlSafeDialog.**TransactionTypesSection**.TransferFundsSection.balancesLoading`

Reply:
Oh, `.TransactionTypesSection.` should not count towards the id of the messages as that's not a component, just a folder that groups a bunch of components

Reply:
Oh I see. I didn't realize we don't include those. Would you mind adding any rules that I am missing for the ids here: https://www.notion.so/colony/Imports-ids-and-displayNames-Naming-Organization-2b5c4f7378364e89a5521f4fa6850f1a 
I don't want to be bugging you in the future with ids 😅 Thank you!

Reply:
DIdn't know this doc existed :eyes: I will